src/loaders/postgres.loader.ts

```diff
@@ -0,0 +1,32 @@
+import { Client } from "pg";
+import { SQL } from "../config/index";
+
+const postgressLoader = async () => {
+  try {
+    // Initialise client
+    const client = new Client(SQL.CONFIG);
+    const connection = await client.connect();
+
+    // Populate table and users
+    await client.query(SQL.QUERIES.CREATE_TABLE);
```

Comment:
Consider exploring "migrations" concept for SQL database.
Essentially, they are perfect for the setup up you're doing here.

Reply:
Thanks, this proved to be very useful (And less bulky than the approach taken here). I've added a migration to create a table and seed users. I've added this to my package.json, so this runs before starting the server (I couldn't seem to find a way to add this to the loaders without running shell commands). Is this okay?

Reply:
"Knex" is actually a. really good solution that enables some pretty complex scenarios.
The alternative is to keep migrations is raw *.sql files and create a bunch of terminal scripts that essentially do the same things as knex does.
Which one to chose, as usual, depends on your requirements (security, packages size, etc).